src/main/js/html.js

```diff
@@ -921,35 +928,44 @@ var backgroundColorAdjustSuffix = "BackgroundColorAdjust";
 
                     if (s * (e.after - frontier) < 0) {
 
-                        pad = Math.ceil(Math.abs(frontier - e.after)) + "px";
```

Comment:
I asked @palemieux why the `Math.ceil` is present here and he thought it was probably a result of implementation differences in browsers, but my recollection is he wasn't sure exactly which implementation caused the problem. I suspect that some browsers round the client rect values to integers and others don't, and depending on the rounding, that could cause a single pixel gap to open up.

Reply:
It's irrelevent, the problem we were really discussing with the ceil problem was due to the padding be applied to both the top and the bottom of adjacent lines, rather than just on the longest of the two, that meant the ceil was bad as it guaranteed an overlap, without the ceil, there's no problem.

Reply:
When I test rendering with this branch on Firefox on Mac OS X I still get a line of overlap between lines, so I'm not sure it's really fixing the issue.

Reply:
f8bd49e fixes this I think